src/components/tutorials-sidebar/components/tutorial-view-sidebar-content/index.tsx

```diff
@@ -64,7 +64,7 @@ function CollectionProgress({ collection }: { collection: Collection }) {
 	 * current collection slug, and list of tutorials in this collection.
 	 */
 	const { completedTutorialCount, tutorialCount, isInProgress } = useMemo(
-		() => parseCollectionProgress(progressData, tutorials, { id, slug }),
+		() => parseCollectionProgress(progressData, tutorials.length, { id, slug }),
```

Comment:
Passing `tutorialCount` rather than all tutorials to the refactored `parseCollectionProgress`.